src/Veldrid/OpenGL/OpenGLExtensions.cs

```diff
@@ -54,6 +54,8 @@ internal OpenGLExtensions(HashSet<string> extensions, GraphicsBackend backend, i
             StorageBuffers = GLVersion(4, 3) || IsExtensionSupported("GL_ARB_shader_storage_buffer_object")
                 || GLESVersion(3, 1);
 
+            CubeMapArrayTexture = GLVersion(1, 3) || GLESVersion(3, 2);
```

Comment:
There appears to be an extension, as well https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_cube_map_array.txt